### PR TITLE
docs: format docs

### PR DESCRIPTION
**Pull request by btea**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: docs/en-US/component/message.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ ElMessage({}, appContext)
 ### Options
 
 | Name                     | Description                                                                                          | Type                                                 | Default |
-|--------------------------|------------------------------------------------------------------------------------------------------|------------------------------------------------------|---------|
+| ------------------------ | ---------------------------------------------------------------------------------------------------- | ---------------------------------------------------- | ------- |
```
Comment (kooriookami) [inline docs/en-US/component/message.md:124]: Why my idea format style is different from yours?
image

Comment (btea) [reply docs/en-US/component/message.md:124]: Do you have any special configuration? There are spaces between the dividing lines of other components. 🤔

Comment (kooriookami) [reply docs/en-US/component/message.md:124]: I use webstorm, may be different.

Comment (btea) [reply docs/en-US/component/message.md:124]: This may be the reason.

**Review by ryuhangyeong — APPROVED** [review]

**Pull request merged**